src/app/components/pages/Board/Card/NewCard.js

```diff
@@ -74,19 +70,15 @@ function NewCard({ client, listId, newPosition }) {
 
   return (
     <form onSubmit={handleSubmit} className="card card--new">
-      <label htmlFor="name">
-        <div>Name</div>
-        <input id="name" value={name} onChange={setName} autoComplete="off" />
-      </label>
-      <label htmlFor="description">
-        <div>Description</div>
-        <input
-          id="description"
-          value={description}
-          onChange={setDescription}
-          autoComplete="off"
-        />
-      </label>
+      {/* eslint-disable-next-line jsx-a11y/label-has-associated-control, jsx-a11y/label-has-for */}
+      <label htmlFor="body">What do you want to say?</label>
```

Comment:
I take it `textarea` can't go inside `label`?

Reply:
It can do, but it doesn't look as good right now - inside the label it isn't full-width.

I find these eslint rules really annoying, an input should either be inside a label or be linked with the `for` attribute so I don't understand why airbnb want both 🤷‍♂  I think I'll just do what their rules say and we can worry about what it looks like when it comes to skinning the app later